examples/step-19/doc/intro.dox

```diff
@@ -0,0 +1,459 @@
+
+<br>
+
+<i>
+This program was contributed by Wolfgang Bangerth, Rene Gassmoeller, and Peter Munch.
+
+Wolfgang Bangerth acknowledges support through NSF
+awards DMS-1821210, EAR-1550901, and OAC-1835673.
+</i>
+
+@note Support for particles exists in deal.II primarily due to the initial
+  efforts of Rene Gassmoeller. Please acknowledge this work by citing
+  the publication @cite GLHPW2018 if you use particle functionality in your
+  own work.
+
+<a name="Intro"></a>
+<h1>Introduction</h1>
+
+The finite element method in general, and deal.II in particular, were invented
+to solve partial differential equations -- in other words, to solve
+[continuum mechanics](https://en.wikipedia.org/wiki/Continuum_mechanics) problems.
+On the other hand, sometimes one wants to solve problems in which it is useful
+to track individual objects ("particles") and how their positions evolve. If
+this simply leads to a set of ordinary differential equations, for example
+if you want to track the positions of the planets in the solar system over
+time, then deal.II is clearly not your right tool. On the other hand, if
```

Comment:
This would really depend on the type of "particle algorithm" you want to implement. A lot of these particle methods (SPH, etc.) use a background grid for neighbour detection (either a tree structure or a flat structure). Consequently, they fall quite nicely in the deal.II framework (especially with the notion of ghost particles). Anyway, just ranting, but deal.II really has nice support even for pure particles method. We use it for our DEM code for example which is purely Lagrangian particle tracking and neighbour detection.

Reply:
Good point -- I added a link.